x-pack/plugins/security_solution/public/detections/components/rules/rule_switch/index.test.tsx

```diff
@@ -21,6 +21,7 @@ import { useAppToastsMock } from '../../../../common/hooks/use_app_toasts.mock';
 jest.mock('../../../../common/hooks/use_app_toasts');
 jest.mock('../../../containers/detection_engine/rules');
 jest.mock('../../../pages/detection_engine/rules/all/rules_table/rules_table_context');
+jest.mock('../../../../common/lib/apm/use_start_transaction');
```

Comment:
The fact that we need to import and mock this hook separately probably tells us that we should be extending `useKibana` hook instead?